ros_arduino_bridge:
  ros__parameters:
    serial_port: "/dev/ttyUSB0"
    baud_rate: 57600
    base_width: 0.215446      # Distance between left/right wheels (meters)
    wheel_radius: 0.085  # Wheel radius (meters)
    encoder_ticks_per_rev: 44  # Encoder ticks per wheel revolution
    odom_frame: "odom"
    base_frame: "base_link"
    imu_frame: "imu_link"
    ultrasonic_left_frame: "ultrasonic_left_link"
    ultrasonic_right_frame: "ultrasonic_right_link"
